src/main/scala/com/microsoft/hyperspace/index/rules/RuleUtils.scala

```diff
@@ -31,9 +31,13 @@ object RuleUtils {
    *
    * @param indexManager indexManager
    * @param plan logical plan
+   * @param hybridScanEnabled HybridScan config.
    * @return indexes built for this plan
    */
-  def getCandidateIndexes(indexManager: IndexManager, plan: LogicalPlan): Seq[IndexLogEntry] = {
+  def getCandidateIndexes(
+      indexManager: IndexManager,
+      plan: LogicalPlan,
+      hybridScanEnabled: Boolean = false): Seq[IndexLogEntry] = {
```

Comment:
nit: can we remove default value? Rationale: this is an user-facing config, which is set to false by default. If we later change the default to true, we need to remember to update this to true as well.

Reply:
It's temporary for this PR - I'll remove the default value assignment & use the config value instead in the next pr.

Reply:
OK. In that case, please leave a review comment on your intention. That helps reviewers.